### PR TITLE
Partner site csv export util

### PR DESCRIPTION
**Pull request by ddellacosta**
[Tracker Card](https://www.pivotaltracker.com/story/show/129880859)

Very simple helper function to dump out a CSV file of all partner sites for import into turbovote partner sites spreadsheet.

### CODE REVIEW

**Comment by ddellacosta** [comment]
@danielglauser Obviously super low priority to review this compared to everything else on our plates now...

File: dev-src/partner_works/dev_util.clj [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  vector in this namespace. Takes a db and csv filename, or uses the
+  default filename 'export-partner-site.csv' if no filename is passed
+  in."
+  ([db]
+   (export-partner-site-csv! db "export-partner-site.csv"))
+  ([db filename]
+   (let [output (export-partner-site-csv* db)]
+     (spit filename (str/join "\n" output)))))
+
+(comment
```
Comment (danielglauser) [inline dev-src/partner_works/dev_util.clj]: Is this meant to be run from the REPL for now, is that why this comment is here?

File: dev-src/partner_works/dev_util.clj [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+       (str/join ",")))
+
+(defn export-partner-site-csv*
+  [db]
+  (->> (partner-site/query-partner-sites db)
+       (sort-by (comp str/lower-case :partner/name))
+       (reduce
+        #(conj %1 (partner-site-csv-row %2)) [(str/join "," csv-header)])))
+
+(defn export-partner-site-csv!
```
Comment (danielglauser) [inline dev-src/partner_works/dev_util.clj]: I don't think this `fn` mutates anything so I don't believe that the `!` is warranted.

**Comment by ddellacosta** [comment]
This is fundamentally broken, needs to be reconsidered. Closing.

**Pull request closed without merging**